unit_tests/test_cluster.py

```diff
@@ -481,6 +481,27 @@ def test_can_get_nodetool_status_typical(self):  # pylint: disable=no-self-use
                            '10.0.198.153': {'state': 'UN', 'load': '?', 'tokens': '256', 'owns': '?',
                                             'host_id': 'fba174cd-917a-40f6-ab62-cc58efaaf301', 'rack': '1a'}}}
 
+    def test_can_get_nodetool_status_typical_with_one_space_after_host_id(self):  # pylint: disable=no-self-use
+        """case for https://github.com/scylladb/scylla-cluster-tests/issues/7274"""
+        resp = "\n".join(["Datacenter: datacenter1",
```

Comment:
probably would be nice to use dedent, but not critical for now

Reply:
probably yes, I don't remember why I chosen this way that time...